CHANGELOG.md

```diff
@@ -66,6 +66,7 @@
 * [ENHANCEMENT] Dashboards: improved resources and networking dashboards to work with read-write deployment mode too. #3497 #3504 #3519 #3531
 * [ENHANCEMENT] Alerts: Added "MimirDistributorForwardingErrorRate" alert, which fires on high error rates in the distributor’s forwarding feature. #3200
 * [ENHANCEMENT] Improve phrasing in Overview dashboard. #3488
+* [BUGFIX] Alerts: Fixed `MimirCompactorSkippedBlocksWithOutOfOrderChunks` alert for when Mimir is deployed in read-write or monolithic modes. #3628
```

Comment:
Please rebase `main` and move this up under "Grafana Mimir - main / unreleased".